mindmeld/models/taggers/embeddings.py

```diff
@@ -192,6 +192,7 @@ def __init__(
         sequence_padding_length,
         token_embedding_dimension=None,
         token_pretrained_embedding_filepath=None,
+        use_padding=True,
```

Comment:
Could we combine `sequence_padding_length ` and `use_padding ` by just stating that if `sequence_padding_length = 0`, then `use_padding => False`?

Reply:
I think it would be counterintuitive to combine them. `sequence_padding_length` is not how much padding we add to the end of each vector, but the total length of the padded sequence. So by that definition, `sequence_padding_length=0` and `use_padding=False` are two different things. In the former you would return a vector of length 0 and the latter you would return a vector of length `token_embedding_dimension * token_sequence_length`